generator/namespaces.py

```diff
@@ -14,12 +15,25 @@
 
 import click
 import yaml
+from google.cloud import storage
 
 from .explores import EXPLORE_TYPES
 from .views import VIEW_TYPES, View
 
 PROBE_INFO_BASE_URI = "https://probeinfo.telemetry.mozilla.org"
 DEFAULT_SPOKE = "looker-spoke-default"
+OPMON_BUCKET_NAME = "operational_monitoring"
+PROD_PROJECT = "moz-fx-data-shared-prod"
+PROJECTS_FOLDER = "projects/"
```

Comment:
One tactic that we've really been leaning on is to expose metadata and configuration files in GIthub instead, where it can be world readable. It does ease debugging and would not require us to have GCP creds here. See if you think it would be worth making that change.

Reply:
Well, right now this is subject to change anyways since we're only temporarily using data from a bucked until we decide what the user interface for creating new operational monitoring projects will look like. It's all set up this way on the bigquery side already now too. So I don't think it's worth the change at the moment but I will keep this in mind as we're designing the new intake process.